modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZCardsParagraphBehavior.php

```diff
@@ -48,6 +48,13 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
       '#description' => $this->t('Select a card style.'),
     ];
 
+    $form['card_clickable'] = [
+      '#title' => $this->t('Clickable cards'),
+      '#type' => 'checkbox',
+      '#default_value' => isset($config['card_clickable']) ? $config['card_clickable'] : '',
```

Comment:
Shouldn't this be a boolean value, now?

```suggestion
      '#default_value' => isset($config['card_clickable']) ? $config['card_clickable'] : FALSE,
```

Reply:
Agreed